README.md

```
@@ -2,313 +2,1435 @@
-* [Kenny Tran Co](https://www.kennytran.co)
-* [Kouceyla Hadji](https://www.behance.net/kossa)
-* [Kumar Abhirup](http://kumar.now.sh)
-* [Kyle Harrison](http://kyle.teamharrison.ca)
-* [Kyle Shook](http://Kyleshook.com)
-* [Katheesh](http://katheesh.gitleaf.com) [console](https://katheesh.github.io/console/)
+
+- [Kaiyu Hsu](https://kyh.io)
+- [Kaleigh Scruggs](http://kaleighscruggs.com)
+- [Kalp Senghani](https://kalpsenghani.com) [👨🏼‍💻 Fullstack Developer || Ai Engineer ]
+- [Kamil Mazurek](https://kamilmazurek.pl)
+- [Kamran Mushtaq](https://kamipresents.com/)[Fullstack Developer]
```

markdownlint-cli2 (0.18.1)

Reversed link syntax
(https://kamipresents.com/)